### PR TITLE
[GEN-491] Set Detail Cleanup

### PR DESCRIPTION
**Pull request by abhipi**
- Created a new backend folder and migrated code to SetDetailLogic.js.
- Frontend components retained in SetDetail, API calls and backend functions were moved.

### CODE REVIEW

File: trk/src/Screens/SetDetail/index.js [hunk 1]
```diff
@@ -20,41 +20,31 @@ function SetDetail(props) {
...
   }, [climbData.images]);
 
   useEffect(() => {
     const fetchTapCount = async () => {
       try {
-        const tapQuerySnapshot = await firestore()
-          .collection('taps')
-          .where('climb', '==', climbData.id)
-          .get();
-        setTapCount(tapQuerySnapshot.size);
```
Comment (VGmt) [inline trk/src/Screens/SetDetail/index.js:57]: thanks for removing.

File: trk/src/Screens/SetDetail/index.js [hunk 2]
```diff
@@ -134,7 +104,7 @@ function SetDetail(props) {
               feedbackList.map((comment, index) => (
                 <View key={index} style={styles.commentContainer}>
                   <Text style={styles.commentText}>{comment.explanation}</Text>
-                  <Text style={styles.commentRating}>{renderRating(comment.rating)}</Text>
+                  <Text style={styles.commentRating}>{() => renderRating(comment.rating)}</Text>
```
Comment (VGmt) [inline trk/src/Screens/SetDetail/index.js:107]: what was the issue here? Was there an issue with rendering?

**Review by VGmt — APPROVED** [review]
Looks great again. Good job Abhishek.  PR is approved – though, I left a question.

File: trk/src/Screens/SetDetail/index.js (hunk 2 above)
Comment (abhipi) [reply trk/src/Screens/SetDetail/index.js:107]: Yupp

**Pull request merged**